core/storage/classifier/gae_models.py

```diff
@@ -65,9 +61,11 @@ class ClassifierTrainingJobModel(base_models.BaseModel):
     # It is incremented by TTL when a job with status NEW is picked up by VM.
     next_scheduled_check_time = datastore_services.DateTimeProperty(
         required=True, indexed=True)
-    # The schema version for the data that is being classified.
-    data_schema_version = (
```

Comment:
Do we need to write a one-Off_job to remove this field from the existing models (if they exist)?

Reply:
Old models do not exist for this.

Reply:
Not required